.github/workflows/macos.yml

```diff
@@ -71,9 +84,38 @@ jobs:
 
       - run: make
 
+      # Launchable CLI requires Python and Java
+      # https://www.launchableinc.com/docs/resources/cli-reference/
+      - name: Set up Python
+        uses: actions/setup-python@871daa956ca9ea99f3c3e30acb424b7960676734 # v5.0.0
+        with:
+          python-version: "3.10"
+      - name: Set up Java
+        uses: actions/setup-java@7a445ee88d4e23b52c33fdc7601e40278616c7f8 # v4.0.0
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Set up Launchable
+        id: setup_launchable
+        run: |
+          set -ex
+          pip install launchable
+          launchable verify
+          : # The build name cannot include a slash, so we replace the string here.
```

Comment:
@junaruga 
FYI: I've added the branch name to the build name to avoid using the same build name as the CI, which runs on other branches. This makes it easier to distinguish each build because we can associate the test results with the individual commits within each branch.

For more information about the build name, please refer to: https://www.launchableinc.com/docs/sending-data-to-launchable/using-the-launchable-cli/recording-builds-with-the-launchable-cli/

Reply:
Since GitHub Actions is defaulted to `bash -e {0}`, `set -x` is enough.

Reply:
Updated
https://github.com/ruby/ruby/pull/9777/files#diff-7b74d73de878e0e53f6768bbf3a3d4b50c3c6f1f16f9b64518690028b8206b1cR121

Reply:
nobu, you are right. The `set -x` is good enough. I confirmed it on my forked repository.